website/content/docs/services/discovery/dns-forwarding/enable.mdx

```diff
@@ -0,0 +1,442 @@
+---
+layout: docs
+page_title: Enable DNS forwarding
+description: ->
+  Learn how to configure different DNS server to perform DNS forwarding to Consul servers.
+---
+
+# Enable DNS forwarding
+
+This topic describes the process to configure different DNS servers to enable DNS forwarding to Consul servers.
+
+The operations performed in this page can be applied on every node where a Consul agent is running.
+
+## Requirements
+
+To enable DNS forwarding, your deployment must have the following:
+
+- A running Consul server instance
+- One or more Consul client nodes with services registered in the Consul catalog
+- One of the following local DNS servers running, `iptables` command available, or macOS running: 
+    - [systemd-resolved](#systemd-resolved)
+    - [BIND](#bind)
+    - [dnsmasq](#dnsmasq)
+    - [Unbound](#unbound)
+    - [iptables](#iptables)
+    - [macOS](#macos)
+
+<Note>
+
+The configuration assumes Consul's DNS server is listening on the loopback interface on the same node of the local DNS server. 
+If Consul is not listening on the loopback IP, replace the references to `localhost` and `120.0.0.1` in the configuration and commands with the appropriate IP address for your environment.
+
+</Note>
+
+## systemm-resolved
+
+[`systemd-resolved`](https://www.freedesktop.org/wiki/Software/systemd/resolved/) is a system service that provides network name resolution to local applications. Is the default local DNS server for many Linux distributions.
+
+You can configure the `systemd-resolved` service to send `.consul` domain queries to Consul by creating a `consul.conf` file located in the `/etc/systemd/resolved.conf.d/` directory.
+
+<Tabs>
+<Tab heading="systemd 245 and older">
+
+Add the following settings to your resolved configuration file:
```

Comment:
```suggestion
Add a `[Resolve]` section to your resolved configuration.
```

For Aimee's reference [Introduce code block with an descriptive, imperative sentence that ends with a period](https://github.com/hashicorp/tutorials/blob/main/STYLE_GUIDE.md#code-blocks).